packages/openbridge-webcomponents/src/navigation-instruments/readout/readout.css

```
@@ -0,0 +1,586 @@
+    .readout-meta-wrapper,
+    .readout-value-wrapper,
+    .readout-value-unit-wrapper,
+    .readout-source-wrapper
+  ) {
+    align-self: center;
+  }
+}
+
+.label {
+  @mixin font-instrument-unit-box;
+  color: var(--element-neutral-color);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)